rest/config.go

```diff
@@ -472,6 +472,29 @@ func (dbConfig *DbConfig) UseXattrs() bool {
 	return base.DefaultUseXattrs
 }
 
+// Create a deepcopy of this DbConfig, or panic.
+// This will only copy all of the _exported_ fields of the DbConfig.
```

Comment:
Calling this 'DeepCopy' and returning an error (that callers can panic on if desired) seems like a better approach than 'MustDeepCopy'.

Reply:
Fixed